Packs/VaronisDataSecurityPlatform/Integrations/VaronisDataSecurityPlatform/VaronisDataSecurityPlatform_description.md

```diff
@@ -0,0 +1,4 @@
+The Varonis Data Security Platform integration allows you to retrieve Varonis DatAlert alerts, create incident and pull activities related to the alerts for conducting investigations.
+
+### Authentication
+In order to use the Varonis DSP package, you can provide / create a user with "Web UI User" role that will be used as integration user.
```

Comment:
Consider adding a link to your web page where users are created (not mandatory)

Reply:
Actually there is no link and it is done via desktop app